### ISSUE TITLE
Two TextInput in a single row

### ISSUE THREAD

**Issue by alokmishra1** [open-olive/loop-development-kit#266]
In whisper can we have two TextInput in a single row. Please provide code for that.

**Comment by alokmishra1** [comment]
I am new to LDK, please help.

**Comment by DanielWagersOlive** [comment]
Hi @alokmishra1!

You should be able to wrap your two `TextInput` components in a `Box` component to get the behavior that you want.  For example:

```js
{
  type: WhisperComponentType.Box,
  justifyContent: JustifyContent.SpaceEvenly,
  direction: Direction.Horizontal,
  children: [
    {
      type: WhisperComponentType.TextInput,
      label: `input 1`,
      onChange: (error: Error, value: string, onClickWhisper: Whisper) => {
        // do nothing.
      },
    },
    {
      type: WhisperComponentType.TextInput,
      label: `input 2`,
      onChange: (error: Error, value: string, onClickWhisper: Whisper) => {
        // do nothing.
      },
    },
  ],
}
```

**Comment by alokmishra1** [comment]
I am getting following error, Please help:
2021-07-07T12:04:52+05:30 ERR Received error calling service invalid arguments: invalid field: Whisper: invalid field: components: missing required field: alignment console=error system=secureloops
2021-07-07T12:04:52+05:30 ERR Unhandled promise rejection GoError: invalid arguments: invalid field: Whisper: invalid field: components: missing required field: alignment console=error system=secureloops
2021-07-07T12:04:52+05:30 INF loop started loopID=88245ffe-a6ec-4ddf-aee6-b9f5f1901440 system=secureloops

**Comment by alokmishra1** [comment]
I am coding in javascript. May be this might be the reason of error.

**Comment by alokmishra1** [comment]
My code:
`{
    type: Box,
    justifyContent: "space_evenly",//JustifyContent.SpaceEvenly"",
    direction: "horizontal",
    children: [
      {
        type: TextInput,
        label: `input 1`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
      {
        type: TextInput,
        label: `input 2`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
    ],
  },`

**Comment by alokmishra1** [comment]
My code:

{
    type: Box,
    justifyContent: "space_evenly",//JustifyContent.SpaceEvenly"",
    direction: "horizontal",
    children: [
      {
        type: TextInput,
        label: `input 1`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
      {
        type: TextInput,
        label: `input 2`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
    ],
  },

**Comment by alokmishra1** [comment]
My code:

`{
    type: Box,
    justifyContent: "space_evenly",//JustifyContent.SpaceEvenly"",
    direction: "horizontal",
    children: [
      {
        type: TextInput,
        label: `input 1`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
      {
        type: TextInput,
        label: `input 2`,
        onChange: (error, value) => {
          // do nothing.
        },
      },
    ],
  },`

**Comment by alokmishra1** [comment]
Resolved: Upgraded LDK to 3.1.0.

**Comment by CandiceCLB** [comment]
I am closing out this issue. @alokmishra1 if you have any other issues, you can either reopen this thread or create a new github issue.
